command/connect/envoy/envoy.go

```diff
@@ -154,26 +154,23 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
-	// First argument needs to be the executable name.
-
-	// TODO(banks): passing config including an ACL token on command line is jank
-	// - this is world readable. It's easiest thing for now. Temp files are kinda
-	// gross in a different way - we can limit to same-user access which is much
-	// better but we are leaving the ACL secret on disk unencrypted for an
-	// uncontrolled amount of time and in a location the operator doesn't even
-	// know about. Envoy doesn't support reading bootstrap from stdin or ENV
-	envoyArgs := []string{binary, "--config-yaml", bootstrapYaml}
-	envoyArgs = append(envoyArgs, passThroughArgs...)
-
-	// Exec
-	err = syscall.Exec(binary, envoyArgs, os.Environ())
-	if err != nil {
-		c.UI.Error("Failed to exec envoy: " + err.Error())
+	err = execEnvoy(binary, passThroughArgs, bootstrapJson)
+	if err == errUnsupportedOS {
+		c.UI.Error("directly running Envoy is only supported on linux and macOS " +
```

Comment:
Is there a recommended wrap length for CLI error output like this?

Reply:
I don't know of one. I think we tend to let the terminal do it's think unless it makes it really hard to read (e.g. pretty printing/tables for dense info).

Reply:
Don't think we have an official one as I mentioned elsewhere - a "screenshot" of the output in the description might be a nice way to judge. I'd personally be tempted to break the second sentence onto a new line but depends what it looks like on the terminal for real.